pkg/tests/handlers/accounts_test.go

```diff
@@ -0,0 +1,40 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/logging"
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/types"
+)
+
+func Test_clusterAuthorizationsBody(t *testing.T) {
+	type args struct {
+		ctx       context.Context
+		clusterID string
+		options   *types.TestOptions
+	}
+	logger, _ := logging.NewGoLoggerBuilder().Build()
+	tests := []struct {
+		name string
+		args args
+	}{
+		// TODO: Add test cases.
+		{"randomize-1", args{context.TODO(), "123", &types.TestOptions{Logger: logger}}},
+		{"randomize-2", args{context.TODO(), "456", &types.TestOptions{Logger: logger}}},
+		{"randomize-3", args{context.TODO(), "1sdjhvcb", &types.TestOptions{Logger: logger}}},
+		{"randomize-4", args{context.TODO(), "haesvsdefhjv", &types.TestOptions{Logger: logger}}},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := clusterAuthorizationsBody(tt.args.ctx, tt.args.clusterID, tt.args.options)
+			if got == nil {
+				t.Errorf("clusterAuthorizationsBody() = %v", got)
+			} else {
+				gotstring := string(got[:])
+				fmt.Println("String =", gotstring)
+			}
```

Comment:
You are not testing anything here. The idea in this second part of the test is we already know the result is not `nil` now we want to check that the `clusterID` we sent to build the body it's actually in the body.